src/book/02-system/01-foundations/05-mathematics.mdx

```diff
@@ -275,3 +277,36 @@ For measurements on the field we use field space. For field coordinates the **x-
 - [Wolfram MathWorld](http://mathworld.wolfram.com/)
 
 - [Eigen Documentation](http://eigen.tuxfamily.org/dox/)
+
+# Interpolation
+
+Let's say we have a collection of data points. We want to find a function that contains all these points. We call this function an interpolant. Often we use a polynomial to describe this function.
+
+## Splines
+
+Splines are used in the [Quintic Walk Engine](/system/subsystems/motion#walk-engine).
+
+Rather than have one global polynomial, it can be useful to use piecewise polynomials. These are all defined locally around the interpolation points.
```

Comment:
```suggestion
Rather than have one global polynomial, it can be useful to use piecewise polynomials, connected at their edges. These so-called [**splines**](https://en.wikipedia.org/wiki/Spline_(mathematics)) are all defined locally around the interpolation points, and can avoid Runge's phenomenon. Splines are used in the [Quintic Walk Engine](/system/subsystems/motion#walk-engine).
```